src/components/global-header/global-header.stories.mdx

```diff
@@ -0,0 +1,225 @@
+import { useState } from "react";
+import { Meta, Canvas, Story } from "@storybook/addon-docs";
+import StyledSystemProps from "../../../.storybook/utils/styled-system-props";
+import NavigationBar from "../navigation-bar";
+import Box from "../box";
+import VerticalDivider from "../vertical-divider";
+import { Menu, MenuFullscreen, MenuItem, MenuSegmentTitle } from "../menu";
+import GlobalHeader from "./global-header.component";
+import useMediaQuery from "../../hooks/useMediaQuery";
+import LinkTo from "@storybook/addon-links/react";
+import carbonLogo from "../../../logo/carbon-logo.png";
+
+<Meta
+  title="Global Header"
+  parameters={{
+    info: { disable: true },
+    chromatic: { disable: true },
+    docs: {
+      inlineStories: false,
+      iframeHeight: 200,
+    },
+  }}
+/>
+
+# Global Header
+
+Global Header is a wrapper component designed for creating site-wide navigation layouts.
+
+## Contents
+
+- [Quick Start](#quick-start)
+- [Examples](#examples)
+- [Props](#props)
+
+## Quick Start
+
+```javascript
+import GlobalHeader from "carbon-react/lib/components/global-header";
+```
+
+## Examples
+
+### Default
+
+Global Header shares similarities with the <LinkTo kind="Navigation Bar" story="default">Navigation Bar component</LinkTo>, but is permanently fixed to the top of the viewport and has a greater z-index than Navigation Bar.
+
+<Canvas>
+  <Story name="default">
+    <GlobalHeader>Example content</GlobalHeader>
+  </Story>
+</Canvas>
+
+### With logo
+
+A custom logo stored as a HTML element or a React component can be rendered by passing it via the `logo` prop. The logo is wrapped in a container that applies style rules for margins and child element height. It is possible to set a custom height for the logo element, but any value that exceeds the height of the Global Header component will be constrained to 40px.
```

Comment:
```suggestion
A custom logo in the form of an HTML element or a React component can be rendered by passing it via the `logo` prop. The logo is wrapped in a container that applies style rules for margins and child element height. It is possible to set a custom height for the logo element, but any value that exceeds the height of the GlobalHeader component will be constrained to 40px.
```

"stored as a" sounds strange to me - I'm not sure the best wording here and am not suggesting "in the form of" is great but I do slightly prefer it, feel free to come up with something better!

It should definitely be `an HTML element` rather than `a HTML element`. (You could simply say "a React element" which covers both HTML elements and React components, but I'm happy to be explicit because in practice the logo will usually be one of these 2 things, and not every developer will instantly know what "React element" means.)

Also, as above, when referencing `GlobalHeader` as a component then I prefer to give the component displayName, which doesn't have a space.

Reply:
Pretty sure it's actually `a` when the following starts with and `h`